### ISSUE TITLE
password Auth failed

### ISSUE THREAD

**Issue by mhmd-81** [mostafamhm/sts#1]
Based on evidences that company gave you the required password but we still get Password authentication failure
check the env file or other files that contain password to the database 
Thanks
